MongoDBLink.m

```diff
@@ -124,6 +126,18 @@
   DatabaseConnection[host, port, JavaNew["com.mongodb.MongoClient", host, port]]
 ]
 
+OpenConnection[host_String, port_Integer, "", _, _]:= OpenConnection[host, port]
```

Comment:
@zbjornson you got the point. When you want anonymous authentication, you can still use OpenConnection with 5 parameters, using user as "". In my case, I read these parameters from a txt file, and I can still read the same number of fields to use OpenConnection, even for anonymous case. Using user -> "", pass -> "" , base -> "". And what matter is the user.